docs/docs/forms.md

```diff
@@ -51,8 +53,10 @@ class NameForm extends React.Component {
   render() {
     return (
       <form onSubmit={this.handleSubmit}>
-        Name:
-        <input type="text" value={this.state.value} onChange={this.handleChange} />
+        <label>
+          Name:
+          <input type="text" value={this.state.value} onChange={this.handleChange} />
+        </label>
```

Comment:
I don't understand how that works even after looking at it. The numbers don't look like they match line numbers...?